packages/turf-line-chunk/index.js

```diff
@@ -0,0 +1,94 @@
+var along = require('@turf/along');
+var lineDistance = require('@turf/line-distance');
+var helpers = require('@turf/helpers');
+var flatten = require('@turf/flatten');
+var linestring = helpers.lineString;
+var point = helpers.point;
+var fc = helpers.featureCollection;
+
+/**
+ * Divides a {@link LineString} into chunks of a specified length.
+ * If the line is shorter than the segment length then the orginal line is returned.
+ * @name lineChunk
+ * @param {Feature<LineString>} line the line to split
+ * @param {Number} segment_length how long to make each segment
+ * @param {String} units can be degrees, radians, miles, or kilometers
+ * @param {Boolean} [asPoints=false] return points instead of linestring segrments
+ * @return {FeatureCollection<LineString>|FeatureCollection<Point>} collection of line segments or points
```

Comment:
You can a feature collection of points back if you pass in the optional 4th param as true. 

Getting line segments back is good, but there are times when you just want the points at which the segments break. Another option might be to create another module which converted a linestring to a set of points (more jsdocs & ts definitions I'll butcher :) )